Demo/Common/Networking/SessionRequest.swift

```diff
@@ -43,10 +43,6 @@ internal struct SessionRequest: APIRequest {
                                   "visa": InstallmentOptions(monthValues: [3, 6, 9], includesRevolving: true)]
         
         try container.encode(installmentOptions, forKey: .installmentOptions)
-        
-        // store payment method required fields
-        try container.encode("askForConsent", forKey: .storePaymentMethodMode)
-        try container.encode("UnscheduledCardOnFile", forKey: .recurringProcessingModel)
```

Comment:
I think `recurringProcessingModel ` is working fine, [its already released](https://docs.adyen.com/api-explorer/Checkout/69/post/sessions#request-recurringProcessingModel)